LICENSE

```diff
@@ -1,201 +1,13 @@
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+Copyright 2016-present WalmartLabs
```

Comment:
Just curious: Is this line based on an "official" template from the legal department or where was this copied from?

Reply:
Not sure if this is the "official" template.  Copied from electrode-io/electrode project.

Reply:
https://github.com/electrode-io/electrode/blob/master/LICENSE is our official open source license.